server/src/main/java/school/journal/controller/ClassAPIController.java

```diff
@@ -76,14 +77,22 @@ public ResponseEntity getOne(HttpServletRequest request, @PathVariable("classId"
     @Qualifier("GenerationService")
     private IGenerationService generationService;
 
-    @RequestMapping(value = "/getMarksForClass", method = RequestMethod.GET)
+    @RequestMapping(value = "/getMarksForClass", method = RequestMethod.GET,produces = "text/csv")
     @ResponseBody
     public ResponseEntity getMarksForClass(HttpServletRequest request, HttpServletResponse response)
             throws ControllerException {
         try {
-//            generationService.generateMarksDocument(DocumentType.CSV, 1, 1);
-            generationService.generateClassPupilListDocument(response.getOutputStream(), DocumentType.PDF, 1);
-        } catch (ServiceException | IOException exc) {}
+            response.setContentType("text/csv");
+            response.setHeader("Content-Disposition", "attachment;filename=mdk.csv");
+//            response.setContentType("text/pdf");
+
+//            generationService.generateMarksDocument(response.getOutputStream(), DocumentType.CSV, 1, 1);
```

Comment:
It is testing endpoint. If u want to try run my generation way, you shouldn't write it.

I checked correctness of generation csv here.